pkg/identity/numericidentity.go

```diff
@@ -365,6 +373,23 @@ func (id NumericIdentity) Uint32() uint32 {
 	return uint32(id)
 }
 
+// GetLocalNodeID returns the configured local node numeric identity that is
+// set in tunnel headers when encapsulating packets originating from the local
+// node.
+func GetLocalNodeID() NumericIdentity {
+	localNodeMutex.Lock()
+	defer localNodeMutex.Unlock()
+	return localNodeIdentity
+}
+
+// SetLocalNodeID sets the local node id.
+// Note that currently changes to the local node id only take effect during agent bootstrap
+func SetLocalNodeID(nodeid uint32) {
+	localNodeMutex.Lock()
+	defer localNodeMutex.Unlock()
+	localNodeIdentity = NumericIdentity(nodeid)
```

Comment:
Given the above change:
```suggestion
	localNodeMutex.Lock()
	defer localNodeMutex.Unlock()
	localNodeIdentity.identity = NumericIdentity(nodeid)
```

Reply:
Done.